### ISSUE TITLE
I want to only 'pre-train'.

### ISSUE THREAD

**Issue by WanzeeCho** [MIC-DKFZ/nnUNet#2233]
Hi
I looked at the document and saw that there was even fine-tuning after pre-training.
But I'm going to use the Teacher-Student model so I want to initialize the teacher model with the weights after pre-training.
Where can I find the weights after pre-training with T1 data?
After initializing the Teacher model with the weights, I want to create pseudo-labels by inserting unlabeled data.

**Comment by TaWald** [comment]
Sounds like you just want to:
1. Normally train your model
2. Use these models to predict unlabeled cases
3. Go back to step 1 with additional, predicted labels

All of this should be trivial if you follow the instructions provided

**Comment by WanzeeCho** [comment]
> Sounds like you just want to:
> 
> 1. Normally train your model
> 2. Use these models to predict unlabeled cases
> 3. Go back to step 1 with additional, predicted labels
> 
> All of this should be trivial if you follow the instructions provided

Thanks for the reply, but I don't think so. 
To explain a little more, we will proceed with the following process. 

1. Pre-train nnU-Net using T1 images and labels
2. Initialize nnU-Net (Teacher Model) with weights of '1'
3. Train '2' on unlabeled T2 data
4. Make pseudo-label from Teacher Model

In this case, what is the path of weights of '1'? Or is there another way to pre-train? In order to proceed according to the provided document, a target dataset is required, but in my case, there is no target dataset. 
To increase the accuracy of the pseudo-label, we want to import weights from the pre-trained model.
